website/README.md

```diff
@@ -0,0 +1,40 @@
+# Lorawan Device Repository Website
+
+An hugo static site generated from the lorawan-devices repository.
+
+## Setup localhost
+
+Run the steps below to run the site at `http://localhost:1313/device-repository/`
+
+Requirements are `nodejs`, `yarn` and `hugo` and `go` , install these yourself.
+
+### Generate content files
+
+`make go.deps`
+`make go.build`
+
+### Install frontend dependencies
+
+`yarn install`
```

Comment:
Getting quite some incorrect and unmet peer dependency warnings. Same with installing the `design` repo